### DISCUSSION TITLE
Where was the world's 🥇 parking meter installed?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#789] [Galaxy Trivia] [answered]
**Bonus:** Where was the 🥈 installed?

**Comment by SANKARAMDAS** [comment]
The world's first parking meter, known as Park-O-Meter No. 1, is installed(on July 16th, 1935.) on the southeast corner of what was then First Street and Robinson Avenue in Oklahoma City, Oklahoma.
Miami, installed in 1936.

**Comment by peterbaccaro** [comment, marked as answer]
The world's first parking meter, known as Park-O-Meter No. 1, is installed on the **southeast corner of what was then First Street and Robinson Avenue** in Oklahoma City, United States. It was installed on 16th July 1935.

The world's second parking meter was also installed on **First Street and Robinson Avenue in Oklahoma City, United States** as meters were placed at 20-foot intervals along the curb.  The city became the pioneer in implementing parking meters as a means to regulate and control parking in urban areas.

https://www.britishparking.co.uk/Timeline/Worlds-first-parking-meter/142630

**Reply by ParanoidUser** [reply]
Good call! Any idea which city after OKC adopted this practice? 🧐

**Reply by peterbaccaro** [reply]
**Tulsa, Oklahoma, United States** was the second city to install parking meters.
